subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/java/JavaLibraryPlugin.java

```diff
@@ -46,16 +50,37 @@ public void execute(CopySpec copy) {
             }
         });
 
+
+        final File dependenciesFile = new File(project.getBuildDir(), "shipkit-dependencies");
+
+        Task copyDepsToFile = TaskMaker.task(project, "createDependenciesFile", CreateDependenciesFile.class, new Action<CreateDependenciesFile>() {
+            @Override
+            public void execute(CreateDependenciesFile task) {
+                task.setDescription("Creates file with resolved runtime dependencies.");
+                task.setOutputFile(dependenciesFile);
+                task.setConfiguration(project.getConfigurations().getByName("default"));
+            }
+        });
+
+        final CopySpec dependencyFile = project.copySpec(new Action<CopySpec>() {
+            public void execute(CopySpec copy) {
+                copy.from(dependenciesFile);
+            }
+        });
+
+
         ((Jar) project.getTasks().getByName("jar")).with(license);
 
         final Jar sourcesJar = project.getTasks().create(SOURCES_JAR_TASK, Jar.class, new Action<Jar>() {
```

Comment:
I'm wondering if we shouldn't add the file to the main jar, too. What do you think?

Reply:
It seems that we don't need in the main jar at the moment, and we can always add it alter if necessary. Do you agree?